pkg/component/component.go

```diff
@@ -963,6 +974,15 @@ func GetComponentFromConfig(localConfig *config.LocalConfigInfo) (Component, err
 	return Component{}, nil
 }
 
+func getPortsProtocol(ports []string) map[string]string {
```

Comment:
maybe a better name for this? `getPortProtocolMapping` and a small comment like `this is a mapping for ports => Protocol`?

Reply:
Ack. Makes sense. Done.